src/domains/canoneunico/03_storage_account.tf

```diff
@@ -0,0 +1,65 @@
+module "cu_sa" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//storage_account?ref=v6.15.2"
```

Comment:
if the storage account is private, you need to create a private endpoint to allow external users to see it, for example when you use the VPN